src/Middleware/JsonError.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/*
+ * This file is part of prooph/proophessor.
+ * (c) 2014-2015 prooph software GmbH <contact@prooph.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Prooph\ProophessorDo\Middleware;
+
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Zend\Stratigility\ErrorMiddlewareInterface;
+
+/**
+ * Error middleware to handle json requests
+ */
+class JsonError implements ErrorMiddlewareInterface
```

Comment:
@prolic Yes it should resolve #11 with this JsonError middleware. /cc @codeliner